setup.py

```diff
@@ -48,22 +48,8 @@
           license=LICENSE,
           url=URL,
           long_description=LONG_DESCRIPTION,
-          packages=['pymc3',
-                    'pymc3.backends',
-                    'pymc3.distributions',
-                    'pymc3.examples',
-                    'pymc3.glm',
-                    'pymc3.step_methods',
-                    'pymc3.step_methods.hmc',
-                    'pymc3.tuning',
-                    'pymc3.tests',
-                    'pymc3.variational',
-                    'pymc3.external',
-                    'pymc3.gp',
-                    'pymc3.plots',
-                    'docs',
-                    '.',
-          ],
+          packages=find_packages(),
```

Comment:
We had this before and it didn't work properly. We should make sure that pip install from git works.